app/routes/api/user/profile/__init__.py

```diff
@@ -0,0 +1,59 @@
+from bson import ObjectId
+from flask import Blueprint, jsonify, request
+import cloudinary
+import cloudinary.uploader
+from app.models.user import User
+
+api_user_profile_bp = Blueprint('api_user_profile', __name__)
+
+def allowed_file(filename):
+    ALLOWED_EXTENSIONS = {'png', 'jpg', 'jpeg', 'gif'}
```

Comment:
define once

Reply:
# Define ALLOWED_EXTENSIONS once at the module level
ALLOWED_EXTENSIONS = {'png', 'jpg', 'jpeg', 'gif'}

def allowed_file(filename):
    # Check if the file has an allowed extension
    return '.' in filename and filename.rsplit('.', 1)[1].lower() in ALLOWED_EXTENSIONS